fuse_constraints/include/fuse_constraints/normal_prior_pose_3d_cost_functor.hpp

```diff
@@ -88,39 +89,42 @@ class NormalPriorPose3DCostFunctor
   bool operator()(const T * const position, const T * const orientation, T * residual) const;
 
 private:
-  fuse_core::Matrix6d A_;
+  fuse_core::MatrixXd A_;
```

Comment:
Why instead here we move to a dynamic size matrix?

Reply:
Same as above